File: 백준/Bronze/31090. 2023은 무엇이 특별할까？/2023은 무엇이 특별할까？.cs
int cnt = int.Parse(Console.ReadLine());

for (int i = 0; i < cnt; i++)
{
    string str = Console.ReadLine();

    int first = int.Parse(str.Substring(2, 2));
    int num = int.Parse(str) + 1;

    Console.WriteLine(num %  first == 0 ? "Good" : "Bye");
}

File: 백준/Bronze/31090. 2023은 무엇이 특별할까？/2023은 무엇이 특별할까？.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int cnt = Integer.parseInt(scanner.nextLine());

        for (int i = 0; i < cnt; i++) {
            String str = scanner.nextLine();

            int first = Integer.parseInt(str.substring(2, 4));
            int num = Integer.parseInt(str) + 1;

            System.out.println(num % first == 0 ? "Good" : "Bye");
        }

        scanner.close();
    }
}
